dataland-frontend/src/components/pages/SearchTaxonomy.vue

```diff
@@ -1,27 +1,39 @@
 <template>
   <AuthenticationWrapper>
     <TheHeader />
-    <TheContent>
+    <TopTabNavigationMenu class="d-fixed-below-header d-fixed-base" :tabs="['EU Taxonomy']" />
+    <!-- This is a spacer div whose only purpose is to ensure that no elements get hidden behind the tab nav -->
+    <div class="h-2rem"></div>
+    <TheContent class="pl-0">
       <div
         class="col-12 bg-white"
-        :class="[searchBarToggled && pageScrolled ? ['d-search-toggle', 'fixed'] : '']"
+        :class="[searchBarToggled && pageScrolled ? ['d-fixed-below-tabnav-toggled', 'd-fixed-base'] : '']"
         ref="searchBarAndIndexTabContainer"
       >
-        <SearchTaxonomyHeader />
+        <SearchTaxonomyHeader class="pl-4" />
         <MarginWrapper>
```

Comment:
Sehr seltsam, warum mir der MarginWrapper hier angezeigt wird, aber in der IDE nicht.
Verstehst du warum GitHub das macht?